### PR TITLE
Bislam

### PR DESCRIPTION
**Pull request by dtf1337**
hi from Colorado

### CODE REVIEW

File: Application/index.js [hunk 1]
```diff
@@ -36,4 +36,8 @@ app.get('$', (req, res) => {
 	res.send('have a nice day!')
   })
 
-app.get('/nope', (req, res) => { res.send('NOPE')})
\ No newline at end of file
+
```
Comment (jinwoov) [inline Application/index.js:39]: Please dont remove other person's work

**Review by jinwoov — CHANGES_REQUESTED** [review]

File: Application/index.js (hunk 1 above)
Comment (dtf1337) [reply Application/index.js:39]: sure, sorry about that.

**Review by dtf1337 — COMMENTED** [review]
le me know if i still need to update anything thank you

**Comment by jinwoov** [comment]
Thank you for adjusting that . Hope you have cool Halloween

**Comment by dtf1337** [comment]
Thank you so much. Sorry about that again. It was great challenge and I
have upgraded my github understanding.

Have a great weekend!

**Pull request merged**